tests/hazmat/primitives/test_serialization.py

```diff
@@ -1355,6 +1355,15 @@ def test_pem_encryption_short_iv(self):
         with pytest.raises(ValueError):
             load_pem_private_key(data, password=b"password")
 
+    def test_dh_key_too_small_modulus_raises_value_error(self):
```

Comment:
What does this test do that the revised `test_load_256bit_key_from_pkcs8` does not?